app/scenes/Login/OAuthScopeHelper.ts

```diff
@@ -0,0 +1,56 @@
+import type { TFunction } from "i18next";
+import capitalize from "lodash/capitalize";
+import uniq from "lodash/uniq";
+import { Scope } from "@shared/types";
+
+export class OAuthScopeHelper {
+  public static normalizeScopes(scopes: string[], t: TFunction): string[] {
+    const methodToReadable = {
+      list: t("read"),
+      info: t("read"),
+      read: t("read"),
+      write: t("write"),
+      create: t("write"),
+      update: t("write"),
+      delete: t("write"),
+      "*": t("manage"),
+    };
```

Comment:
It seems like the naming here differs slightly from what the backend calls it in methodToScope? Specifically the `"*": "manage"` entry

https://github.com/outline/outline/pull/8884/files#diff-f59462f1f54f9b78195ce4dc8b29cc24a94e74043d8959f8f69764d08590b543R9-R23

Reply:
"Manage" is kind of a catch all, maybe too ambiguous and should be changed to "Read & Write"?